test/mason/publish-tests/publishFork.chpl

```diff
@@ -0,0 +1,13 @@
+use MasonPublish;
+use MasonUtils;
+use FileSystem;
+use Spawn;
+
+proc main() {
+  var cmd = 'cloneMasonReg("test", here.cwd())';
```

Comment:
This isnt testing the mason call. It is testing the ``runWithStatus`` function. You will need to call ``cloneMasonReg``